quick_classifier.py

```diff
@@ -79,7 +107,7 @@ def bottom_k_scores(scores, k):
     output_idx = 0
     output_list = []
     output_batch = 500
-    header = 'bibcode,title,abstract,keywords,scores,threshold'
+    header = 'bibcode,uat_branch'
```

Comment:
Add UAT number

Reply:
Header for output filename changed to 'bibcode,uat_branch,uat_numner'.